src/commands/demo-store.js

```diff
@@ -312,27 +313,52 @@ ${chalk.dim(manifest.description)}`)
       const env = await envWriter.create(`${this.getDestinationDirectory(manifest)}${sep}.env`)
 
       Object.entries(manifest.dotenv).forEach(([key, value]) => {
-        // Check for an injection point of chec keys
-        const matches = typeof value === 'string' && value.match(/^%chec_([ps])key%$/)
-
-        if (!matches) {
-          env.set(key, value)
-          return
+        try {
+          return env.set(key, this.substituteEnvVars(value, key))
+        } catch (error) {
+          this.log(error.message)
```

Comment:
```suggestion
          this.error(error.message)
```

maybe?

Reply:
I considered this but it seems like that API is more for when the command can't continue. I've used `log` because the store can still install, it just isn't able to set an env var properly

Reply:
Yeah OK. I think in a lot of cases that would result in a broken install, so maybe it would be better to stop the process here (runtime error vs call time)

Reply:
I can do that, but I haven't implemented any clean-up for errors. Do you think it should show what's wrong with all the "dotenv" settings, then exit out? Or do you think it should clean up the downloaded directory?

Reply:
I'd say leave it there and exit out showing the problem

Reply:
Ok I can probably whip that together. Give me a few more mins